src/containers/Comments/Comment.js

```diff
@@ -91,50 +95,45 @@ export const Comment = (props) => {
     setHasExcerpt(e.nativeEvent.lines.length > NUM_OF_LINES)
   }, [])
 
-  function handleOptions(subcomment) {
-    if (subcomment !== undefined){
-      commentBeingProcessed.current = subcomment
-    } else {
-      commentBeingProcessed.current = comment
-    }
-    props.setDeleteModalItems(commentDeleteModalItems)
-    props.setMenuItems(commentMenuItems)
+  function handleOptions(commentInstance) {
+    props.setDeleteModalItems(getCommentDeleteModalItems(commentInstance))
+    props.setMenuItems(getCommentMenuItems(commentInstance))
     props.toggleMenu()
   }
 
-  const handleRate = async(ratingComment, value) => {
+  const handleRate = async(commentInstance, value) => {
```

Comment:
I just renamed this to commentInstance everywhere to be consistent and distinguish it from comment in state.